pkg/bloomcompactor/bloomcompactor.go

```diff
@@ -211,13 +211,16 @@ func (c *Compactor) running(ctx context.Context) error {
 	for {
 		select {
 		case <-ticker.C:
+			start := time.Now()
 			c.metrics.compactionRunsStarted.Inc()
 			if err := c.runCompaction(ctx); err != nil {
-				c.metrics.compactionRunsFailed.Inc()
+				c.metrics.compactionRunsCompleted.WithLabelValues("fail").Inc()
```

Comment:
Do we have common name for the label value in the failure case? I think we should try to be consistent with existing label names.

Reply:
Good call. I see we use "failure" for failed statuses. I've added constants for both statuses.